app/server.js

```diff
@@ -41,7 +41,7 @@ server.use(function(req, res, next) {
 });
 
 // routes ======================================================================
-server.use(subdomain('api', require('./routes/api'))); // load our api routes and configure them to use the api subdomain
+server.use(subdomain('api', require('./routes/api')(passport))); // load our api routes and configure them to use the api subdomain
```

Comment:
This seems less than ideal, but I don't have a better solution so I guess this LGTM for now